diff --git a/node_modules/expo-iap/ios/ExpoIapModule.swift b/node_modules/expo-iap/ios/ExpoIapModule.swift
index c6e7a2e..dc3f83f 100644
--- a/node_modules/expo-iap/ios/ExpoIapModule.swift
+++ b/node_modules/expo-iap/ios/ExpoIapModule.swift
@@ -14,9 +14,9 @@ public final class ExpoIapModule: Module {
     nonisolated public func definition() -> ModuleDefinition {
         Name("ExpoIap")
 
-        Constant("ERROR_CODES") {
-            OpenIapSerialization.errorCodes()
-        }
+        Constants([
+            "ERROR_CODES": OpenIapSerialization.errorCodes()
+        ])
 
         Events(
             OpenIapEvent.purchaseUpdated.rawValue,
diff --git a/node_modules/expo-iap/ios/onside/OnsideIapModule.swift b/node_modules/expo-iap/ios/onside/OnsideIapModule.swift
index cf6f7d7..6ac7cbc 100644
--- a/node_modules/expo-iap/ios/onside/OnsideIapModule.swift
+++ b/node_modules/expo-iap/ios/onside/OnsideIapModule.swift
@@ -670,13 +670,13 @@ public final class ExpoIapOnsideModule: Module {
     nonisolated public func definition() -> ModuleDefinition {
         Name("ExpoIapOnside")
 
-        Constant("ERROR_CODES") {
-            OpenIapSerialization.errorCodes()
-        }
+        Constants([
+            "ERROR_CODES": OpenIapSerialization.errorCodes()
+        ])
 
-        Constant("IS_ONSIDE_KIT_INSTALLED_IOS") {
-            false
-        }
+        Constants([
+            "IS_ONSIDE_KIT_INSTALLED_IOS": false
+        ])
 
         Events(
             OnsideEvent.purchaseUpdated.rawValue,
